IhorKremenskiy/HW2/books.js

```diff
@@ -0,0 +1,97 @@
+
+let bookName = prompt("Please enter a book name"); // Asking a user to enter a book name
+let bookAuthor = prompt("Please enter a book author"); // Asking a user to enter a book author
+let bookYear = prompt("Please enter a book publication year"); // Asking a user to enter a book publication year
+let bookPages = prompt("Please enter a book pages count"); // Asking a user to enter a book pages count
+
+
+
+// When all questions answered, the user should be asked to _confirm_ book save:
+// - Would you like to add "%bookName%" to the library?
+// If the user chooses "OK" he should be notified with the message:
+// - "%bookName%" by  %bookAuthor% successfully saved to the library!
+// (example: "The Adventures of Tom Sawyer" by Mark Twain successfully saved to the library!)
+
+
+// If the user chooses "Cancel" program should stop.
+
+if (confirm (`Would you like to add ${bookName} to the library?`)){
+    alert (`${bookName} by ${bookAuthor} successfully saved to the library`)
+
+} else close
```

Comment:
**close** - The expression is not a function call.
There is a close function in javascript required to close a browser tab, but we don't need to close a tab, we have to stop the program immediately if the user chooses "Cancel", but tab should still be alive.

With your implementation, even if you will call a close function, the tab will be closed only after 
bookResult.getBookSize() and bookResult.getPublicationCentury() functions calls